q2_moshpit/kraken2/classification.py

```diff
@@ -50,7 +51,7 @@ def _construct_output_paths(
 def _classify_kraken2(
         seqs, common_args
 ) -> (Kraken2ReportDirectoryFormat, Kraken2OutputDirectoryFormat):
-    if isinstance(seqs, MAGSequencesDirFmt):
+    if type(seqs) is MAGSequencesDirFmt or type(seqs) is ContigSequencesDirFmt:
```

Comment:
You could also do `isinstance(seqs, (MAGSequencesDirFmt, ContigSequencesDirFmt))` for simplicity (or was the intention to do a strict type checking?)

Reply:
I didn't know you could use `isinstance` that way, thanks!